lib/Backend/Commands.hs

```diff
@@ -350,11 +349,7 @@ buildCopyOperations oldBackend newBackend oldQPath@(QualifiedPath oldBackendName
         newEntry <- entry & E.path . Path.entryPathParentDir %%~ \parentDir ->
           case Path.replacePathPrefix oldPath newPath parentDir of
             Just newParentDir -> pure newParentDir
-            Nothing -> throw $ OtherError $ unlinesF @_ @Builder
-              [ "Internal error:"
-              , "Expected path: '" <> pretty (entry ^. E.path) <> "'"
-              , "To have the prefix: '" <> pretty oldPath <> "'"
-              ]
+            Nothing -> throw $ InternalCommandsError (EntryPathDoesntHavePrefix (entry ^. E.path) oldPath)
```

Comment:
this line is a tad over 80, it could be broken up

Reply:
Done